### ISSUE TITLE
newrelic_synthetics_multilocation_alert_condition validate function for violation_time_limit_seconds is too strict

### ISSUE THREAD

**Issue by acookin** [newrelic/terraform-provider-newrelic#2475]
Hi there,

Thank you for opening an issue. In order to better assist you with your issue, we kindly ask to follow the template format and instructions. Please note that we try to keep the Terraform issue tracker reserved for **bug reports** and **feature requests** only. General usage questions submitted as issues will be closed and redirected to New Relic's Explorers Hub https://discuss.newrelic.com/c/build-on-new-relic/developer-toolkit.

## Please include the following with your bug report

> :warning: **Important:** Failure to include the following, such as omitting the Terraform configuration in question, may delay resolving the issue.

- [x] Your New Relic `provider` [configuration](#terraform-configuration-files) (sensitive details redacted)
- [x] A list of [affected resources](#affected-resources) and/or data sources
- [x] The [configuration](#terraform-configuration-files) of the resources and/or data sources related to the bug report (i.e. from the list mentioned above)
- [x] Description of the [current behavior](#actual-behavior) (the bug)
- [x] Description of the [expected behavior](#expected-behavior)
- [x] Any related [log output](#debug-output)

### Terraform Version
`1.5.1`

### Affected Resource(s)

- `newrelic_synthetics_multilocation_alert_condition`

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration
> Please include your `provider` configuration (sensitive details redacted) as well as the configuration of the resources and/or data sources related to the bug report.
```hcl
terraform {
  required_version = "~> 1.5.1"
  required_providers {
    newrelic = {
      source = "newrelic/newrelic"
      version = "~> 3.27.1"
    }
  }
}

provider "newrelic" {
  account_id = <READACTED>
}

resource "newrelic_synthetics_multilocation_alert_condition" "this" {
  policy_id = 1

  name    = "condition name"
  enabled = true
  violation_time_limit_seconds = 259200

  entities = ["monitoridredacted"]
  critical {
    threshold = 2
  }
}
```

### Actual Behavior
Got an error when running terraform plan:

```
│ Error: expected violation_time_limit_seconds to be one of [0 3600 7200 14400 28800 43200 86400], got 259200
```

### Expected Behavior
I'm importing this resource in to terraform, and the resource is currently configured to close after 3 days (e.g. 259200 seconds). I would expect to be able to configure this in terraform. I'm currently needing to ignore this value entirely in the terraform lifecycle.

### Steps to Reproduce
1. `terraform plan` the above resource

### Important Factoids

It would appear the validation function just needs amending

https://github.com/newrelic/terraform-provider-newrelic/blob/main/newrelic/resource_newrelic_synthetics_multilocation_alert_condition.go#L82-L86

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by pranav-new-relic** [comment]
Hi @acookin, thanks for reporting this. We'll take a look at this issue soon.

**Comment by pranav-new-relic** [comment]
A fix to this has been addressed via #2493 - which will be out with the next release of the Terraform Provider, soon this week or early next week.

**Comment by pranav-new-relic** [comment]
Released with [v3.27.4](https://registry.terraform.io/providers/newrelic/newrelic/3.27.4) of the Terraform Provider; thank you for reporting this.
